Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This file is part of Flask-RateLimiter
+# Copyright (C) 2015 CERN.
+#
+# Flask-RateLimiter is free software; you can redistribute it and/or
+# modify it under the terms of the Revised BSD License; see LICENSE
+# file for more details.
+
+FROM python:2.7
+ADD . /code
+WORKDIR /code
+RUN pip install pep257
+RUN pip install sphinx
+RUN pip install redis
+RUN python setup.py develop
```

Comment:
I see, I misunderstood "pip for everything".  Yes, I'll repush shortly, adding `(PR #16)` and `Reviewed-by` you.